### PR TITLE
Add Fernando Migliorini Tenório

### PR DESCRIPTION
**Pull request by dotenorio**

### CODE REVIEW

File: data/speakers.ts [hunk 1]
```diff
@@ -6042,6 +6042,26 @@ const speakers: Speaker[] = [
       city: 'New York',
     },
   },
+  {
+    objectID: 274,
```
Comment (yakovlevyuri) [inline data/speakers.ts]: @dotenorio I'm changing ID to smt more human-readable, I'm adding detail page to the app, so your page is going to be like `confcitizens.com/dotenorio`. Or you can choose whatever you want username :)

```suggestion
    objectID: 'dotenorio',
```

File: data/speakers.ts [hunk 2]
```diff
@@ -6042,6 +6042,26 @@ const speakers: Speaker[] = [
...
+    tags: ['Open Source', 'Community', 'Node.js', 'JavaScript', 'Now', 'Zeit', 'Serverless', 'Angular'],
+    email: null,
+    website: 'https://dotenorio.com/',
+    social: {
+      twitter: 'dotenorio',
+      github: 'dotenorio',
+      blog: 'https://blog.dotenorio.com/',
+      linkedin: null,
+    },
+    talks: [],
```
Comment (yakovlevyuri) [inline data/speakers.ts]: If you already have some talks, could you please add them here? See `Sara Vieira` as an example. Thank you very much!

Comment (dotenorio) [reply data/speakers.ts]: I'll do it, tks

**Comment by dotenorio** [comment]
Now i believe it1s correct.. Tks for your comments :D

**Pull request merged**